ci-operator/jobs/openshift/origin/openshift-origin-periodics.yaml

```diff
@@ -62,6 +62,64 @@ periodics:
     - configMap:
         name: prow-job-cluster-launch-installer-src
       name: job-definition
+- agent: kubernetes
+  cron: ""
+  decorate: true
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: origin
+  interval: 24h
+  name: periodic-ci-e2e-aws-multitenant
```

Comment:
This should be structured like a release periodic, not a post submit periodic.  We want to run tests on releases, not tests on branches.

Add this to openshift-release-release-4.2-periodics.yaml as `release-....<same_as_others>...-e2e-aws-network-multitenant-4.2`

Reply:
You'll want to set the RELEASE_IMAGE_LATEST to `registry.svc.ci.openshift.org/ocp/release:4.2`